runner/src/main.rs

```diff
@@ -79,17 +92,21 @@ async fn main() -> Result<()> {
     runs.add(1, &[KeyValue::new("command", args.command.name())]);
 
     info!(?args.command, ?args.otlp_endpoint, "starting runner");
-    let success = match args.command {
+    let success = match args.command.clone() {
         Command::Bootstrap(opts) => bootstrap(opts).await?,
         Command::Simulate(opts) => simulate(opts).await?,
+        Command::GenerateLoad(opts) => simulate_load(opts).await?,
         Command::Noop => CommandResult::Success,
     };
-
-    // Flush traces and metrics before shutdown
-    shutdown_tracer_provider();
-    metrics_controller.force_flush()?;
-    drop(metrics_controller);
-    shutdown_meter_provider();
+    if !matches!(args.command, Command::GenerateLoad(_)) {
```

Comment:
rather than clone the command and do this match again, you could just check `if metrics_controller.is_some()`.. if you don't like overloading the meaning that way, you could return a tuple above with a ` let (metrics_controller, shutdown_tracing) = match .. { (None, false) } ...` or something